### PR TITLE
Fix some error and warning in the NPC sample folder

### PR DESCRIPTION
**Pull request by sader1992**
* **Addressed Issue(s)**: none

* **Server Mode**: both

* **Description of Pull Request**: Fix some error and warning in the NPC sample folder

to test
```
npc: doc/sample/bank_test.txt
npc: doc/sample/basejob_baseclass_upper.txt
npc: doc/sample/checkoption.txt
npc: doc/sample/delitem2.txt
npc: doc/sample/getequipcardid.txt
npc: doc/sample/getequipid.txt
npc: doc/sample/getiteminfo.txt
npc: doc/sample/getmonsterinfo.txt
npc: doc/sample/gstorage_test.txt
npc: doc/sample/inarray.txt
npc: doc/sample/instancing.txt
npc: doc/sample/localized_npc.txt
npc: doc/sample/navigate.txt
npc: doc/sample/npc_dynamic_shop.txt
npc: doc/sample/npc_extend_shop.txt
npc: doc/sample/npc_live_dialogues.txt
npc: doc/sample/npc_shop_test.txt
npc: doc/sample/npc_test_array.txt
npc: doc/sample/npc_test_chat.txt
npc: doc/sample/npc_test_duplicate.txt
npc: doc/sample/npc_test_func.txt
npc: doc/sample/npc_test_npctimer.txt
npc: doc/sample/npc_test_npctimer2.txt
npc: doc/sample/npc_test_pcre.txt
npc: doc/sample/npc_test_quest.txt
npc: doc/sample/npc_test_setitemx.txt
npc: doc/sample/npc_test_setmapflag.txt
npc: doc/sample/npc_test_skill.txt
npc: doc/sample/npc_test_time.txt
npc: doc/sample/randomopt.txt
```

### CODE REVIEW

File: doc/sample/npc_test_duplicate.txt [hunk 1]
```diff
@@ -12,7 +12,7 @@
 //=  'OnInit' loads the middle Poring last, for some reason.
 //============================================================
 
--	script	Test Script	-1,1,1,{
```
Comment (sader1992) [inline doc/sample/npc_test_duplicate.txt]: i don't know if this should work
however it case crash for the mapserver if there is no npc object in the map for OnTouch npc

**Comment by sader1992** [comment]
https://github.com/rathena/rathena/pull/3458/files#r214432683

**Comment by Atemo** [comment]
Anything left?

**Comment by sader1992** [comment]
there is some more i will checkem out today

**Comment by sader1992** [comment]
the only one left is localized_npc.txt as i did not understand what this npc for , maybe i am using it wrong so i think this pr is done for now

**Comment by Atemo** [comment]
Thanks !

**Pull request merged**